packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -77,6 +78,7 @@ class DateTimeUtils {
    * @param {object} momentDate - The moment object to synchronize the date and time.
    * @param {string} time - The time to synchronize with the moment object.
    * @param {boolean} hasSeconds - If true seconds will be processed
+   * @param {string} timeZone - Time zone value for the moment object.
```

Comment:
Looks like this param is not needed.

Reply:
removed here e3a49adabb7322a4265611a27d15cd77a0110374